docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,46 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE time_zone_id
+    SET TIME ZONE LOCAL
+
+Description
+-----------
+
+Sets the current default time zone displacement for the current SQL-session.
+
+
+Examples
+--------
+
+The following example sets the default time zone to the original time zone
+of the current SQL-session::
+
+    SET TIME ZONE LOCAL;
+
+A custom time zone for the current SQL-session can be set by specifying the
+time zone id as a string identifier::
+
+    SET TIME ZONE 'America/Los_Angeles';
+    SET TIME ZONE '-08:00';
+
+or as an interval::
+
+    SET TIME ZONE INTERVAL -'08:00' HOUR TO MINUTE;
```

Comment:
How about the limitation of the interval being in the range [-14:00, +14:00]? Is it respected? It should be mentioned in the documentation.